examples/fluent-bit/send-fb-internal-metrics-to-cw/README.md

```diff
@@ -50,6 +50,20 @@ This example contains the following:
 
 This is necessary because currently, the [Fluent Bit metrics](https://docs.fluentbit.io/manual/pipeline/inputs/fluentbit-metrics) and the [Prometheus Metrics](https://docs.fluentbit.io/manual/pipeline/inputs/prometheus-scrape-metrics) inputs do not emit their data as logs, and use a separate metric pipeline that most Fluent Bit plugins do not support. However, if we use the exec input to curl the metrics, then the outputted prometheus data is turned into logs which we can parse and process easily. In the future, this experience may be improved.
 
+For a quick setup, use the built-in plugin metrics configuration file available in AWS for Fluent Bit 2.X.Y+. This built-in configuration means that you do not have to build a custom Fluent Bit image. However, please note that this built-in configuration enables all plugin metrics (not just output plugin metrics).
```

Comment:
Must wait till we merge and release: https://github.com/aws/aws-for-fluent-bit/pull/485